File: src/org/usfirst/frc/team6325/robot/commands/Lift/JoystickBackIntake.java
package org.usfirst.frc.team6325.robot.commands.Lift;

import edu.wpi.first.wpilibj.command.Command;

import org.usfirst.frc.team6325.robot.Robot;


/**
 *
 */
public class JoystickBackIntake extends Command {
	
	public JoystickBackIntake() {
		// Use requires() here to declare subsystem dependencies
		requires(Robot.backBelts);
	
	}

	// Called just before this Command runs the first time
	@Override
	protected void initialize() {
		
	}

	// Called repeatedly when this Command is scheduled to run
	@Override
	protected void execute() {
		Robot.backBelts.moveBackBelts(-Robot.oi.xboxController.getRawAxis(5));
	}

	// Make this return true when this Command no longer needs to run execute()
	@Override
	protected boolean isFinished() {
		return false;
	}

	// Called once after isFinished returns true
	@Override
	protected void end() {
		
	}

	// Called when another command which requires one or more of the same
	// subsystems is scheduled to run
	@Override
	protected void interrupted() {
		
	}
}

File: src/main/java/org/usfirst/frc/team6325/robot/commands/Lift/JoystickBackIntake.kt
package org.usfirst.frc.team6325.robot.commands.Lift

import org.sertain.command.*
import org.usfirst.frc.team6325.robot.Polybius
import org.usfirst.frc.team6325.robot.IDs


class JoystickBackIntake : Command()
{
    fun JoystickBackIntake() = requires(Polybius.backBelts)

    override fun execute() = Polybius.backBelts.moveBackBelts(-Polybius.oi.xboxController.getRawAxis(IDs.xboxIDs.get("Right-Joystick-Y-Axis")))

    override fun isCompleted() = return false
}